Add exclusivity question to Grantee FAQ (#496)
OpenSats/website: data/pages/faq-grantees.mdx

commitment, [reach out to us][c]. We can discuss adjusting your grant based on a
smaller time commitment.

### Does OpenSats allow funding by multiple organizations at the same time?

> Yes, but you must disclose the terms of your other funding (e.g. grants) to OpenSats.

Reviewer: nit, maybe nicer-sounding
```suggestion
Yes, provided you disclose the terms of your other funding (e.g. grants) to OpenSats.
```
